frontend/package.json

```diff
@@ -5,7 +5,6 @@
   "dependencies": {
     "@emotion/react": "^11.9.0",
     "@emotion/styled": "^11.8.1",
-    "@mui/icons-material": "^5.6.0",
```

Comment:
Honestly saying, you need somehow to setup this package. Maybe try `yarn add @mui/material-icons --network-timeout 500000` (setup big timeout for installation).